### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by DanielFrankDK** [Taiizor/ReaLTaiizor#184] [Ideas]
Hey.

I like the idea of a library, that makes it easy to style your application.

However, I think more people would be prone to use it, if there was some documentation on how to even use it.

Having to go through every single control and their properties manually, is really just a deal-breaker for me, no matter how awesome this project is and how much effort you've put into it.

Please, consider writing some documentation.

**Comment by Taiizor** [comment]
You are right about the lack of documentation, but unfortunately, I don't have the time to dedicate to it. For a long time now, I've only been able to address small issues.

WinForms has become an outdated technology. Technologies like WPF, UWP, and MAUI offer much more modern and aesthetically pleasing UI/UX libraries. They may seem more complex compared to WinForms, but technology has evolved in that direction.

Additionally, the source code for every visual element you see in the project is available within the repository. If you want to check out an application that includes a general usage example, [click here](https://github.com/Taiizor/ReaLTaiizor/tree/develop/demo/ReaLTaiizor.UI).
